steps/codecov/3.3.0/step.yml

```diff
@@ -0,0 +1,69 @@
+title: Codecov
+summary: Upload your code coverage files to Codecov.io
+description: |-
+  Reduce technical debt with visualized test performance,
+  faster code reviews and workflow enhancements.
+  Now you can upload your code coverage files to Codecov every time you kick off a build!
+website: https://codecov.io
+source_code_url: https://github.com/codecov/codecov-bitrise
+support_url: https://community.codecov.io
+published_at: 2023-06-27T09:35:26.180619-04:00
+source:
+  git: https://github.com/codecov/codecov-bitrise.git
+  commit: 133598e022aca08c75e8af1d32a4a794010336c0
+type_tags:
+- test
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: true
+inputs:
+- CODECOV_TOKEN: null
```

Comment:
Step input key should use lower snake case style: https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#step-input-keys-and-values

Reply:
got it, thanks!